webapp/src/components/Countdown/Countdown.css

```
@@ -0,0 +1,34 @@
+.Countdown-col{
+  display: inline-block;
+}
+
+.Countdown-col-element, .banner .Countdown-col-element {
+  display: inline-block;
+  margin: 0 10px;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.Countdown-col-element.dots {
+  margin: 0;
+}
+
+.Countdown-col-element p, .banner .Countdown-col-element p{
+  font-size: 2em;
+  margin-bottom: 0;
+}
+
+Countdown-col-element span {
+  text-transform: capitalize
+}
+
+@media(min-width: 768px) {
+  .Countdown-col-element{
+    margin: 0 20px;
+  }
+
+  .Countdown-col-element p{
```

Can I ask for a prettier format to this file?

for example:
```suggestion
  .Countdown-col-element p {
```